lib/api/core/entrypoints/embedded/protocols/socketio.js

```diff
@@ -26,11 +26,13 @@ const
   debug = require('../../../../../kuzzleDebug')('kuzzle:entry-point:protocols:socketio'),
   ClientConnection = require('../clientConnection'),
   Protocol = require('./protocol'),
-  Request = require('kuzzle-common-objects').Request,
   {
-    KuzzleError,
-    InternalError: KuzzleInternalError
-  } = require('kuzzle-common-objects').errors;
+    Request,
+    errors: {
+      KuzzleError,
+      InternalError
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this override of "InternalError". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS2424)